Nodejs/Product/Nodejs/Repl/NpmReplCommand.cs

```diff
@@ -76,19 +76,9 @@ class NpmReplCommand : IReplCommand {
                     continue;
                 }
 
-                EnvDTE.Properties properties = dteProject.Properties;
-                if (dteProject.Properties == null) {
-                    continue;
-                }
-
                 string projectName = dteProject.Name;
-                EnvDTE.Property projectHome = properties.Item("ProjectHome");
-                if (projectHome == null || projectName == null) {
-                    continue;
-                }
-
-                var projectDirectory = projectHome.Value as string;
-                if (projectDirectory != null) {
+                string projectDirectory = Path.GetDirectoryName(dteProject.FullName);
```

Comment:
this won't work when the project file is in a different location than the directory itself.